kernel/build

```diff
@@ -17,10 +17,12 @@ if [ "$(date +'%V%y') $KERNEL" != "$(cat kern-dl-date-$BASEBOARD 2>/dev/null)" ]
 	echo "Downloading kernel for $BASEBOARD/$TARGET"
```

Comment:
changes of this whole file have to be removed as soon as cadmium 5.14 kernel is working again

Reply:
Make it work.

Reply:
hold my beer

Reply:
you are drinking beer? my turn!

Reply:
well it works now for me